src/components/Element/component.tsx

```diff
@@ -101,6 +116,25 @@ class Component <P extends ElementArgs, S> extends React.Component <P, S> {
                     // @ts-ignore
                     this.element[prop] = this.props[prop];
                 }
+            } else if (prop === 'class') {
+                let classProp: Array<string> | string = this.props[prop] ?? [];
+                if (!Array.isArray(classProp)) {
+                    classProp = [classProp];
+                }
+                classProp.forEach((cls: string) => {
+                    this.element.class.add(cls);
+                });
+                if (!this.class) {
```

Comment:
Is it possible for `this.class` to be undefined here?

Reply:
No this was unnecessary, I've removed it.